app/controllers/dashboard/associated_users_controller.rb

```diff
@@ -67,7 +67,7 @@ def new
   end
 
   def create
-    creator = AssociatedUserCreator.new(project_role_params)
+    creator = AssociatedUserCreator.new(project_role_params, current_user)
```

Comment:
In Dashboard, this should be `@user` instead of `current_user`. Not 100% sure if `current_user` won't work, but all other dashboard controllers and views use `@user`